modules/page/html/render/tag.js

```diff
@@ -0,0 +1,57 @@
+const { computed, map, h } = require('mutant')
+const nest = require('depnest')
+const TagHelper = require('scuttle-tag')
+
+exports.needs = nest({
+  'about.obs.name': 'first',
+  'keys.sync.id': 'first',
+  'message.html.render': 'first',
+  'message.obs.get': 'first',
+  'sbot.obs.connection': 'first',
+  'tag.html.tag': 'first'
+})
+
+exports.gives = nest('page.html.render')
+
+exports.create = function (api) {
+  return nest('page.html.render', function channel (path) {
+    if (!path.startsWith('/tags')) return
+    const urlId = path.substr(6)
+    const currentTagId = urlId && decodeURIComponent(urlId)
+    const myId = api.keys.sync.id()
+    const ScuttleTag = TagHelper(api.sbot.obs.connection)
+
+    const tags = map(ScuttleTag.obs.allTagsFrom(myId), tagId => ({
+      tagId,
+      tagName: api.about.obs.name(tagId)
+    }))
+    const name = currentTagId ? api.about.obs.name(currentTagId) : 'Select A Tag'
+    const tagMessages =
+      currentTagId ? map(ScuttleTag.obs.messagesTaggedByWith(myId, currentTagId), msgId =>
```

Comment:
_personal preference_ :open_hands: 

I think I'd prefer this api to be `ScuttleTag.obs.taggedMessages({ tagger:myId, tagId: currentTagId  })`

then if you omit `tagger` or `tagId` ScuttleTag handles what to do (which this current pattern suggests it might already be doing?). It could also handle the getting of the messages on its side to save some mess this side